src/components/Button.tsx

```diff
@@ -74,11 +74,9 @@ export const Button = (props: Props) => {
       {props.icon && <Icon {...props.icon} />}
       <P
         style={{
-          color: 'inherit',
-          fontSize: 'inherit',
-          flexGrow: 1,
           marginLeft: props.icon === undefined ? 0 : 8,
         }}
+        cssStyles={getCSSStyle('primaryLabel')}
```

Comment:
i think we should call this "label" since all buttons use it, not just primary.

Reply:
Agreed. I will refactor that in the next iteration, okay?